app/assets/stylesheets/users.scss

```diff
@@ -0,0 +1,44 @@
+.container {
+  gap: 2rem;
+  padding: 2rem;
+

```

Comment:
- [ ] OPTIONAL
One of your pages does not match the given [wireframe templates](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md), considering the route `http://[::1]:3000/users/1/posts/1`. Kindly note we as much as possible want our design to closely match the given template. Kindly endeavor to improve upon your styling to match the given template.

**The original wireframe:**
![image](https://user-images.githubusercontent.com/12745474/166861891-4052da6d-6836-4389-9a88-fe908c35a195.png)


**Your design:**
![image](https://user-images.githubusercontent.com/12745474/166861929-6334bf98-13a1-47aa-ad26-f614bd475f4d.png)